Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -265,6 +263,18 @@
             "timeContextFromParameter": "TimeRange",
             "queryType": 0,
             "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "6aa9fe7f-8c73-4871-8b9f-c15014832721",
+            "version": "KqlParameterItem/1.0",
+            "name": "barChartMode",
+            "type": 1,
+            "description": "Get bar Chart mode",
+            "isRequired": true,
+            "query": "print iff('{UsageSplitBy}' == '<unset>' and '{isTimeRange}' == 'true', 'TimeBar', iff('{UsageSplitBy}' == '<unset>' and '{isTimeRange}' == 'false', 'ByValueBar', 'SplitBar'))",
```

Comment:
instead of using a query that prints something, look at using the "criteria" parameter, which lets you do if/then/else style things

Reply:
will update at another pr